doc/development/08_oauth.md

```diff
@@ -30,6 +30,7 @@ Different external applications need different types of data about their users.
 | `people`        | people API endpoints                      |
 | `invoices`      | invoice API endpoints                     |
 | `mailing_lists` | mailing list API endpoints                |
+| `nextcloud`     | maps roles to nextcloud-groups            |
```

Comment:
```suggestion
| `nextcloud`     | nextcloud groups based on user's roles    |
```
A mapping would intuitively mean a hash to me, but this scope just outputs a flat list of groups.